colabs/Similarity_tests.ipynb

```diff
@@ -0,0 +1,1113 @@
+        "                                                  \"no_nominations_oscars_norm\", \"no_nominations_globes_norm\",\n",
+        "                                                  \"no_nominations_emmy_norm\", \"no_films_norm\",\n",
+        "                                                  \"average_films_rating_norm\", \"genres_code\", \"types_code\"])\n",
+        "validate_data = pd.read_parquet(VALIDATE_DATA, columns=[\"nconst\", \"tconst\", \"category\", \"primaryName\", \"knownForTitles\",\n",
+        "                                                  \"no_nominations_oscars_norm\", \"no_nominations_globes_norm\",\n",
+        "                                                  \"no_nominations_emmy_norm\", \"no_films_norm\",\n",
+        "                                                  \"average_films_rating_norm\", \"genres_code\", \"types_code\"])"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "AMzXW0oIGAqT",
+        "outputId": "e3c12643-f3e5-4c3d-e754-5a677c31b2d3"
+      },
+      "execution_count": 7,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "CPU times: user 33.3 s, sys: 7.24 s, total: 40.5 s\n",
+            "Wall time: 53.3 s\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Porównywanie dwóch aktorów"
+      ],
+      "metadata": {
+        "id": "r8q5uNEDV57T"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Wyciągnięcie pierwszego aktora"
+      ],
+      "metadata": {
+        "id": "Oum5fGAmGpzt"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Weź id aktora ze strony imdb. Wpisujemy imię i nazwisko aktora w wyszukiwarkę na imdb i kopiujemy jego id z linku, np. dla Natalie Portman id bierzemy z linku: https://www.imdb.com/name/nm0000204/?ref_=nv_sr_srsg_0"
+      ],
+      "metadata": {
+        "id": "qfBx7BfWcUdR"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "actor1_id = \"nm0000204\""
+      ],
+      "metadata": {
+        "id": "4E_wuUsfIfVM"
+      },
+      "execution_count": 8,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "if (train_data['nconst'] == actor1_id).any():\n",
+        "    actor1_data = find_actor(train_data,  actor1_id)\n",
+        "    display(pd.DataFrame(actor1_data))\n",
+        "    actor1_data = prepare_pandas_row(actor1_data)\n",
+        "elif (validate_data['nconst'] == actor1_id).any():\n",
+        "    actor1_data = find_actor(validate_data,  actor1_id)\n",
+        "    display(pd.DataFrame(actor1_data))\n",
+        "    actor1_data = prepare_pandas_row(actor1_data)\n",
+        "else:\n",
+        "    print(\"Aktora nie ma w danych treningowych i walidacyjnych. Wybierz innego aktora.\")"
```

Comment:
ja bym to zrefactorował do jakiejś funkcji, używamy tego kilka razy i nie będzie tyle powtarzanego kodu się pojawiało

Reply:
przeniosłam